airbyte_cdk/sources/file_based/stream/abstract_file_based_stream.py

```diff
@@ -179,7 +179,6 @@ def record_passes_validation_policy(self, record: Mapping[str, Any]) -> bool:
             )
 
     @cached_property
-    @deprecated("Deprecated as of CDK version 3.7.0.")
```

Comment:
and here i was praying that i'd never have to see the words `availability_strategy` ever again